### DISCUSSION TITLE
What's #1116 for?

### DISCUSSION THREAD

**Discussion by donaldli2020** [Prodigy-Hacking/ProdigyMathGameHacking#1117] [Questions] [answered]
What does the message really mean in #1116?

**Comment by PatheticMustan** [comment, marked as answer]
it's just so i can refernce it in commits. Kinda like a tracker for what hacks we've re-written

**Reply by hugo1234567890123** [reply]
hi pathetic must man  do you remember me?

**Reply by hugo1234567890123** [reply]
A long time ago

**Reply by PatheticMustan** [reply]
>a long time ago
>generic username, not really identifiable
>joined github this month

?????
